vector/src/main/java/fr/gouv/tchap/features/home/room/list/TchapRoomListDisplayModeFilter.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package fr.gouv.tchap.features.home.room.list
+
+import im.vector.app.features.home.RoomListDisplayMode
+import io.reactivex.functions.Predicate
+import org.matrix.android.sdk.api.session.room.model.Membership
+import org.matrix.android.sdk.api.session.room.model.RoomSummary
+
+class TchapRoomListDisplayModeFilter(private val displayMode: RoomListDisplayMode) : Predicate<RoomSummary> {
```

Comment:
I don't think this class `TchapRoomListDisplayModeFilter` will be used later.
Can you confirm?
(Is it possible to keep using the existing RoomListDisplayModeFilter?)

Reply:
Yes I confirm this, if we have only one representation of rooms list in Tchap.

Reply:
TchapRoomListDisplayModeFilter has been replaced by RoomListDisplayModeFilter